### PR TITLE
[Intel MKL] Fix memory leak in dnnl 0.21.2

### PR DESCRIPTION
**Pull request by guizili0**

### CODE REVIEW

File: tensorflow/core/kernels/mkl_fused_batch_norm_op.cc [hunk 1]
```diff
@@ -718,9 +724,9 @@ class MklFusedBatchNormOp : public OpKernel {
         std::memcpy(batch_variance_data, variance_data, depth_ * sizeof(U));
       }
     } catch (mkldnn::error& e) {
-      string error_msg = "Status: " + std::to_string(e.status) +
-                         ", message: " + string(e.message) + ", in file " +
-                         string(__FILE__) + ":" + std::to_string(__LINE__);
+      string error_msg = "Status: " + std::to_string(e.status) + ", message: " +
+                         string(e.message) + ", in file " + string(__FILE__) +
+                         ":" + std::to_string(__LINE__);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_batch_norm_op.cc]: This reformatting shouldn't happen. Please make sure you're using `clang-format` 8.0.1 and reformat. (Or manually revert this for now.)

File: tensorflow/core/kernels/mkl_fused_batch_norm_op.cc [hunk 2]
```diff
@@ -1064,9 +1070,9 @@ class MklFusedBatchNormGradOp : public OpKernel {
                   reinterpret_cast<char*>(diff_weights_data + depth_),
                   depth_ * sizeof(U));
     } catch (mkldnn::error& e) {
-      string error_msg = "Status: " + std::to_string(e.status) +
-                         ", message: " + string(e.message) + ", in file " +
-                         string(__FILE__) + ":" + std::to_string(__LINE__);
+      string error_msg = "Status: " + std::to_string(e.status) + ", message: " +
+                         string(e.message) + ", in file " + string(__FILE__) +
+                         ":" + std::to_string(__LINE__);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_batch_norm_op.cc]: Please revert this reformatting as well.

File: tensorflow/core/util/mkl_util.h [hunk 3]
```diff
@@ -732,9 +732,9 @@ inline Status ConvertMklToTF(OpKernelContext* context,
     }
     return Status::OK();
   } catch (mkldnn::error& e) {
-    string error_msg = "Status: " + std::to_string(e.status) +
-                       ", message: " + string(e.message) + ", in file " +
-                       string(__FILE__) + ":" + std::to_string(__LINE__);
+    string error_msg = "Status: " + std::to_string(e.status) + ", message: " +
+                       string(e.message) + ", in file " + string(__FILE__) +
+                       ":" + std::to_string(__LINE__);
```
Comment (penpornk) [inline tensorflow/core/util/mkl_util.h]: This reformatting shouldn't happen.

File: tensorflow/core/util/mkl_util.h [hunk 4]
```diff
@@ -1259,8 +1259,8 @@ inline Status CreateBlockedMemDescHelper(const memory::dims& dim,
   } catch (mkldnn::error& e) {
     return Status(error::Code::INTERNAL,
                   tensorflow::strings::StrCat(
-                      "Failed to create blocked memory descriptor.",
-                      "Status: ", e.status, ", message: ", e.message));
+                      "Failed to create blocked memory descriptor.", "Status: ",
+                      e.status, ", message: ", e.message));
```
Comment (penpornk) [inline tensorflow/core/util/mkl_util.h]: This reformatting shouldn't happen.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! Some of the formattings don't match ours. Could you please try using `clang-format` 8.0.1? Thank you!

**Comment by guizili0** [comment]
@penpornk Thanks for the comments, I have update the format by using clang-format 8.0.1.
BTW, is this version the minimal requirement in the format check? Thank you!

**Review by penpornk — APPROVED** [review]
Thank you for the quick fixes! 
I'm not sure if a newer version will have any issues. Please keep using version 8.0.1 for now.

**Comment by guizili0** [comment]
@penpornk Thank you, will use the clang-format 8.0.1 to do the format check, thanks.

**Pull request merged**